kyaml/yaml/rnode.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strconv"
 	"strings"
 
+	"sigs.k8s.io/kustomize/api/internal/utils"
```

Comment:
@natasha41575 `sigs.k8s.io/kustomize/api/internal/utils` is already imported then why error is occuring saying 
```
 yaml/rnode.go:15:2: no required module provides package sigs.k8s.io/kustomize/api/internal/utils; to add it:
	go get sigs.k8s.io/kustomize/api/internal/utils
```

Reply:
You have to run the `go get` command so that it can automatically update the go.mod and go.sum files.